velox/dwio/common/CMakeLists.txt

```diff
@@ -54,6 +52,8 @@ add_library(
   TypeWithId.cpp
   WriterFactory.cpp)
 
+target_include_directories(velox_dwio_common PRIVATE ${Protobuf_INCLUDE_DIRS})
```

Comment:
@kgpai This PR https://github.com/prestodb/presto/pull/20370/files fixes the header leak.
We don't need protobuf for Prestissimo with this change.

Reply:
@majetideepak [#20370 ](https://github.com/prestodb/presto/pull/20370/files) is still in draft so I will refrain from reviwing it till you feel its ready. Looking at #5781

Reply:
@majetideepak @kgpai It turns out this is needed when pinning the Protobuf version to 3.21 instead of 3.21.4. If the there is no exact match the system protobuf is used. And the line 23 include makes it so that protobuf header 23.x is included because that version resides in the location.

This is a Velox only change, these changes together with the Protobuf pinning PR should fix the build issues for Velox when multiple Protobuf versions are present.

It seems like a safe change for Velox?